apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -143,6 +144,7 @@ describe('MultiPayment', () => {
 
                     const integrationClient = await makeClientWithNewRegisteredAndLoggedInUser()
                     await createTestAcquiringIntegrationAccessRight(admin, acquiringIntegration, integrationClient.user)
+                    // TODO(toplenboren): Fix this test
```

Comment:
New TODO format should include Jira task number

Reply:
resolved 👍